<!DOCTYPE html>
<html lang="en">

    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
        <link rel="stylesheet" href="https://www.w3schools.com/lib/w3-theme-black.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
        <!-- http://getbootstrap.com/docs/4.5/ -->
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
        <link href="/static/styles.css" rel="stylesheet">
        <!-- http://getbootstrap.com/docs/4.5/ -->
        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>

    <style>
 .navbar .navbar-nav .nav-link {
    color: #FFFFFF;
    font-size: 1.5em;
}
.navbar .navbar-nav .nav-link:hover{
    color: #808080;
}
@media only screen and (min-width: 960px) {
  .navbar {
    flex-direction: column;
  }
  .navbar .navbar-nav {
    border-bottom: 2px solid #000;
  }
  .navbar .nav-item {
    padding: 0.5em 1em;
  }
}
.navbar .navbar-nav {
  margin-top: 0.5em;
  border-top: 2px solid #000;
}
p {
text-align: justify;
  font-size: 1.5em;
}
.center {
  display: block;
  margin-left: auto;
  margin-right: auto;
  width: 90%;
}
h2 {
  padding: 10px;
  border: 5px solid gray;
  width: 60%;
  display: block;
  margin-left: auto;
  margin-right: auto;
}

    </style>
 </head>


<nav class="navbar navbar-expand-lg navbar-light"  style="background-color: #48546c;">
    <a class="navbar-brand" href="#"><img src="/static/logo_outline.png" width="400"></a>
    <p style="color: #bcbcc3" ;>Rapid Evaluation of Experimental RNA Selection/Evolution</p>

    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#myNavbarToggler13"
        aria-controls="myNavbarToggler13" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="myNavbarToggler13">
        <ul class="navbar-nav mx-auto">
            <li class="nav-item">
                <a class="nav-link" href="/home">Home</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="/choose_rounds">Analyze</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="/help">Help</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="/tutorial">Tutorial</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="/resources">Other Resources</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="/contact">Contact</a>
            </li>
        </ul>
    </div>
</nav>


<div >
    <div>
            <br>
            <h1 align = "center" STYLE = #96D1CD >HELP</h1>
            <br>
            <h2 class="b1" align="center" >
            COMPATIBILITY
            <br>
            <img src="/static/compat.png" alt="User Image" width="900">



            </p>

            </h2>
            <br>

            <h2 class="b1" align="center" >
            USAGE
            <p STYLE= "font-size:.8em">
            The REVERSE pipeline allows users to rapidly identify, cluster, and analyze nucleic acid sequences from large (and sometimes difficult to interpret for the uninitiated) sequencing files to generate usable results with a few clicks. REVERSE takes as input multiple FASTQ files (which, if too large, can be reduced in size before analysis) from high-throughput sequencing data obtained from different rounds of in vitro selection/evolution experiments. As an alternative to experimentalists spending time on laborious computer programming analyses to identify functional sequences to be used for downstream characterization, REVERSE identifies the most abundant sequences/sequence families, graphically compares and tracks the emergence of each selected sequence across various rounds of selection, and enables their nucleotide level analysis in a matter of minutes. REVERSE also generates publication quality figures wherever relevant.
        Although REVERSE is developed for analyzing HTS data from nucleic acid selection/evolution experiments, it can be used to analyze data generated by other sequencing experiments. For example, REVERSE can be used to identify conserved and variable residues in viral genomes.
            </p>

            </h2>
            <br>

            <h2 align="center" >

            FUNCTIONALITY
            <br>
            <img src="/static/pipeline_ppt.png" alt="User Image" width="900">

            </h2>
            <br>

            <h2 align="center" >
            OUTPUTS
            <p STYLE= "font-size:.8em">
            1) <b>Important sequences and selection statistics.</b>The first level of analysis is to detect enrichment in specific sequences (active molecules) in the course of selection and identify those sequences with their relative abundances. REVERSE generates downloadable files (either download as CSV or right click and select "Save Image As") with the list of unique sequences and their reads after various pre-processing and analytical steps. It also tabulates important information about the selection such as total number of unique sequences, number of sequences that pass the quality threshold, and the sequence diversity in each round.
            </p>
            <p STYLE= "font-size:.8em">2) <b>Figures.</b> Creating figures from sequence data in a selection/evolution experiment can be challenging due to the vast size of the datasets and the difficulties associated with data processing. REVERSE generates customizable figures for the results that would be useful for most selection/evolution experiment. For example, REVERSE counts the number of reads of the N (user defined) most abundant sequences in each round of selection plots their abundance in each round allowing to the user to identify the round in which functional enrichment occurred. A lack of significant enrichment would usually indicate an unsuccessful selection.
            </p>
            <p STYLE= "font-size:.8em">
            3) <b>Raw data.</b>
            While REVERSE incorporates some custom features into its figure generation tools, it also provides the raw data which can be used to create similar figures with modification in the software of choice.
            </p>
            </h2>

            <br>

            <h2 align="center" >
            SOURCE CODE
            <p STYLE= "font-size:.8em">
            All code used in the creation of the REVERSE webserver can be found on
            <a href="https://github.com/szostaklab/REVERSE/tree/main/mysite">GitHub</a>.


            </p>
            </h2>


</div>
</div>
    </body>
    <br>
<img src="/static/copy.png" width="200">

</html>