<!DOCTYPE html>
<html lang="en">
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link rel="stylesheet" href="https://www.w3schools.com/lib/w3-theme-black.css">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<style>
html,body,h1,h2,h3,h4,h5,h6 {font-family: "Roboto", sans-serif;}
    .header {
    margin-top: 20px;
    }
    .b1 {
    margin-top: 20px;
    }
.tooltip {
    position: relative;
    display: inline-block;
    border-bottom: 1px dotted black;
    font-family: sans-serif;
    font-weight: bold;
    text-align: center;
    font-size: 0.8em;
    line-height: 0.8em;
    border-radius: 50%;
    margin-left: 6px;
    padding: 0.13em 0.2em 0.09em 0.2em;
    color: inherit;
    border: 1px solid;
    text-decoration: none;
}

.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  border-radius: 6px;
  padding: 5px 0;
  font-size: 15px;

  /* Position the tooltip */
  position: absolute;
  z-index: 1;
}

.tooltip:hover .tooltiptext {
    visibility: visible;
}
h2 {
    /* margin: 0 auto; or use auto left/right margin instead of justify-content center */
    background-color:white;
    border-radius: 25px;
    border: 2px solid #96D1CD;
    padding: 20px;

}
.center_logo {
  position: absolute;
  left: 50%;
  top: 14%;
  transform: translate(-50%, -50%);
    }
    .center {
  position: absolute;
  left: 50%;
  top: 68%;
  transform: translate(-50%, -50%);
    }
    .header_use {
  text-align: center;
  background: white;
  color: black;
  font-size: 15px;
  padding-bottom: 2px;
  padding-top: 17px;
  top: 10%;
    }
    </style>
<body>

<!-- Navbar -->
<div class="w3-top">
  <div class="w3-bar w3-theme w3-top w3-left-align w3-large">
    <a class="w3-bar-item w3-button w3-right w3-hide-large w3-hover-white w3-large w3-theme-l1" href="javascript:void(0)" onclick="w3_open()"><i class="fa fa-bars"></i></a>
    <a href="/home" class="w3-bar-item w3-button w3-hide-small w3-hover-white">Home</a>
    <a href="/choose_rounds" class="w3-bar-item w3-button w3-hide-small w3-hover-white">Analyze</a>
    <a href="/help" class="w3-bar-item w3-button w3-hide-small w3-hover-white">Help</a>
    <a href="/tutorial" class="w3-bar-item w3-button w3-hide-small w3-hover-white">Tutorial</a>
    <a href="/resources" class="w3-bar-item w3-button w3-hide-small w3-hide-medium w3-hover-white">Other Resources</a>
    <a href="/contact" class="w3-bar-item w3-button w3-hide-small w3-hide-medium w3-hover-white">Contact</a>

  </div>
</div>


<div class="w3-row w3-padding-64">
    <div class="w3-twothird w3-container">

    <div class="header_use">
    </div>

            <h1 STYLE = #96D1CD >HELP</h1>
            <h2 class="b1" align="center" >
            USAGE
            <p STYLE= "font-size:.6em">
            The REVERSE pipeline allows users to rapidly identify, cluster, and analyze nucleic acid sequences from large (and sometimes difficult to interpret for the uninitiated) sequencing files to generate usable results with a few clicks. REVERSE takes as input multiple FASTQ files from high-throughput sequencing (HTS) data obtained from different rounds of in vitro selection/evolution experiments. As an alternative to experimentalists spending time on laborious computer programming analyses to identify functional sequences to be used for downstream characterization, REVERSE identifies the most abundant sequences/sequence families, graphically compares and tracks the emergence of each selected sequence across various rounds of selection, and enables their nucleotide level analysis in a matter of minutes. REVERSE also generates publication quality figures wherever relevant.
        Although REVERSE is developed for analyzing HTS data from nucleic acid selection/evolution experiments, it can be used to analyze data generated by other sequencing experiments. For example, REVERSE can be used to identify conserved and variable residues in viral genomes.
            </p>

            </h2>

            <h2 align="center" >

            FUNCTIONALITY

            <p STYLE= "font-size:1em">STEP 1: Enter Number of Rounds</p>
            <p STYLE= "font-size:1em">STEP 2: Enter Quality Cutoff</p>
            <p STYLE= "font-size:1em">STEP 3: Choose Number of Sequences to Analyze. Upload Selection Round Zipped FASTQ Files.</p>
            <p STYLE= "font-size:1em">STEP 4</p>
            <p STYLE= "font-size:.6em">Step 4a: Download your (up to 10,000) most abundant high quality sequences (identify region of interest and need for the reverse complement)</p>
            <p STYLE= "font-size:.6em">STEP 4b: Trimmer: Enter the first and last positions of the sequence of interest. If unsure, enter start = 1, end = N, since your sequences are N nucleotides long. This function removes sequencing handles, barcodes, and constant regions (if any).</p>
            <p STYLE= "font-size:.6em">STEP 4c: Reverse Complementer: View your sequences above to determine if this is needed.</p>
            <p STYLE= "font-size:.6em">STEP 4d: Pipeline Chooser: This option determines the analysis options you are given in the following page.</p>
            <p STYLE= "font-size:1em">Processing</p>
            <p STYLE= "font-size:.7em">Pipeline 1: Analyze All Individual Sequences</p>
            <p STYLE= "font-size:.55em">Download Preprocessed Data: Quality controlled, Trimmed, Converted to the Reverse Complement if desired. Using the parameters you entered on the previous screen, generate a .CSV file for each round of selection containing only the sequences (and their number of counts) with quality scores that exceeded the threshold, which are trimmed to the inputted length. If you selected the reverse complement option, these sequences will be reverse complements.</p>
            <p STYLE= "font-size:.55em">Selection Statistics: Generate a table containing the total number and fraction of unique and high quality sequences.</p>
            <p STYLE= "font-size:.55em">Sequence Finder : Identify the most abundant sequences in the final round of your selection.</p>
            <p STYLE= "font-size:.55em">Sequence Tracker: Generate figures that calculate the abundance of each of the most abundant sequences in each round of selection. Use this to understand when enrichment occured and how much it varies by round.</p>
            <p STYLE= "font-size:.55em">Sequence Motif Finder: Given a string of nucleotides that you want to track in your sequences through the rounds of selection (or the emergence of a known motif through evolution), this generates a figure with the string's fraction abundance in each round. Use A, C, G, T, R (purine), Y (pyrimidine), and N (any) only.</p>
            <p STYLE= "font-size:.55em">Mutational Intermediates Finder: Given two sequences from the data, output all intermediates between them also in the data. Particularly useful for in vitro evolution experiments which evolve one phenotype to another. Determine the evolutionary trajectory connecting two functions.</p>

            <p STYLE= "font-size:.7em">Pipeline 2: Cluster Sequences and Analyze Sequence Clusters</p>
            <p STYLE= "font-size:.55em">Cluster Sequences: Input maximum number of clusters, the optimal number of clusters will be calculated. It will generate a .CSV file with the sequences, their counts, and their assigned cluster.</p>
            <p STYLE= "font-size:.55em">Clusters Peak Finder: This will output the most abundant cluster peak sequences (a larger input number will take longer to run).</p>
            <p STYLE= "font-size:.55em">Cluster Tracker: Generate figures that calculate the fraction abundance of the peak sequence of each of the most abundant clusters in each round of selection. Use this to understand when enrichment occured and how much it varies by round.</p>
            <p STYLE= "font-size:.55em">Conservation Finder: This will generate a heatmap showing the distribution of nucleotides at each position of a specified cluster from the final round.</p>

            </h2>

            <h2 align="center" >
            OUTPUTS
            <p STYLE= "font-size:.6em">
            1) <b>Important sequences and selection statistics.</b>The first level of analysis is to detect enrichment in specific sequences (active molecules) in the course of selection and identify those sequences with their relative abundances. REVERSE generates downloadable files with the list of unique sequences and their reads after various pre-processing and analytical steps. It also tabulates important information about the selection such as total number of unique sequences, number of sequences that pass the quality threshold, and the sequence diversity in each round.
            </p>
            <p STYLE= "font-size:.6em">2) <b>Figures.</b> Creating figures from sequence data in a selection/evolution experiment can be challenging due to the vast size of the datasets and the difficulties associated with data processing. REVERSE generates customizable figures for the results that would be useful for most selection/evolution experiment. For example, REVERSE counts the number of reads of the N (user defined) most abundant sequences in each round of selection plots their abundance in each round allowing to the user to identify the round in which functional enrichment occurred. A lack of significant enrichment would usually indicate an unsuccessful selection.
            </p>
            <p STYLE= "font-size:.6em">
            3) <b>Raw data.</b>
            While REVERSE incorporates some custom features into its figure generation tools, it also provides the raw data which can be used to create similar figures with modification in the software of choice.
            </p>
            </h2>


</div>
</div>
    </body>
</html>